multichain-testing/scripts/fast-usdc.ts

```diff
@@ -0,0 +1,224 @@
+#!/usr/bin/env -S node --import ts-blank-space/register
+import '@endo/init';
+import { parseArgs } from 'node:util';
+import type { ExecutionContext } from 'ava';
+import { commonSetup } from '../test/support.js';
+import { makeFeedPolicy, oracleMnemonics } from '../test/fast-usdc/config.js';
+import { makeDenomTools } from '../tools/asset-info.js';
+import { makeDoOffer } from '../tools/e2e-tools.js';
+import { AmountMath, type Brand } from '@agoric/ertp';
+import { divideBy } from '@agoric/zoe/src/contractSupport/ratio.js';
+import type { USDCProposalShapes } from '@agoric/fast-usdc/src/pool-share-math.js';
+import type { PoolMetrics } from '@agoric/fast-usdc/src/types.js';
+
+const contractName = 'fastUsdc';
+const contractBuilder =
+  '../packages/builders/scripts/fast-usdc/init-fast-usdc.js';
+
+const mockT = {
+  log: console.log,
+  is: (condition: unknown, expected: unknown, message: string) => {
+    if (condition !== expected) {
+      throw new Error(
+        `Condition: ${message} failed. Expected ${expected} got ${condition}.`,
+      );
+    }
+  },
+} as unknown as ExecutionContext;
+
+// from ../test/fast-usdc/fast-usdc.test.ts
+type VStorageClient = Awaited<ReturnType<typeof commonSetup>>['vstorageClient'];
+const agoricNamesQ = (vsc: VStorageClient) =>
+  harden({
+    brands: <K extends AssetKind>(_assetKind: K) =>
+      vsc
+        .queryData('published.agoricNames.brand')
+        .then(pairs => Object.fromEntries(pairs) as Record<string, Brand<K>>),
+  });
+
+const fastLPQ = (vsc: VStorageClient) =>
+  harden({
+    metrics: () =>
+      vsc.queryData(`published.fastUsdc.poolMetrics`) as Promise<PoolMetrics>,
+    info: () =>
+      vsc.queryData(`published.${contractName}`) as Promise<{
+        poolAccount: string;
+        settlementAccount: string;
+      }>,
+  });
+
+const parseCommandLine = () => {
+  const { values, positionals } = parseArgs({
+    options: {
+      oracle: {
+        type: 'string',
+      },
+      mnemonic: {
+        type: 'string',
+      },
+      help: {
+        type: 'boolean',
+      },
+    },
+    allowPositionals: true,
+  });
+
+  if (values.help) {
+    console.log(`
+Usage:
```

Comment:
please move this Usage string constant to the top of the file.

I was going to ask for a file comment, but this is in some ways better.

The file name gave very little clue about what it's for too... though now that I see what it's for, I suppose that name is not bad. But maybe `fast-usdc-tool.ts` would be better.